src/graphics/program-library.js

```diff
@@ -35,11 +34,14 @@ class ProgramLibrary {
         this._defaultStdMatOption = {};
         this._defaultStdMatOptionMin = {};
 
-        const m = new StandardMaterial();
-        m.shaderOptBuilder.updateRef(
-            this._defaultStdMatOption, {}, m, null, [], SHADER_FORWARD, null);
-        m.shaderOptBuilder.updateMinRef(
-            this._defaultStdMatOptionMin, {}, m, null, [], SHADER_SHADOW, null);
+        standardMaterial.shaderOptBuilder.updateRef(
+            this._defaultStdMatOption, {}, standardMaterial, null, [], SHADER_FORWARD, null);
+        standardMaterial.shaderOptBuilder.updateMinRef(
+            this._defaultStdMatOptionMin, {}, standardMaterial, null, [], SHADER_SHADOW, null);
```

Comment:
I feel like this stuff should be factored out of the program library code. But for now, it's fine.

Reply:
agreed